apps/background/app.lisp

```diff
@@ -0,0 +1,25 @@
+;imports
+(import 'sys/lisp.inc)
+(import 'class/lisp.inc)
+(import 'gui/lisp.inc)
+
+(defq id t index 0
+	frames (map (lambda (_) (canvas-load (cat "apps/background/orangewp1280x1080.cpm") load_flag_shared)) (range 1 2)))
+
```

Comment:
Here you don't need to load a set of frames, you can just say:
(defq id t index 0
     canvas (canvas-load (cat "apps/background/orangewp1280x1080.cpm") load_flag_shared))

And maybe not even load_flag_shared, as who else is going to share this image ? so 0 would be fine. Loading it with load_flag_shared flag will make it get cached in memory in case others use it, and it will stay there even if nobody is using it !

Reply:
I've yet to add the ability to stretch draw a canvas, that's something I will work on over this weekend as your going to need it !

Reply:
Implementing stretch/shrink would be great. Scale, center, and zoom are great for just about any graphics applications at some point. if their implementation is fairly trivial. At this point little is trivial for me. :) 

I kept the map lambda around to implement dynamic wallpapers. Like the list for on-start apps, to allow transition of wallpapers over time (for instance one during day one during night or one every five minutes, whatever the case may be.